civihr_employee_portal/views/templates/views-view-table--Tasks--block.tpl.php

```diff
@@ -65,6 +65,7 @@
 
 function _get_task_filter_by_date($date) {
     $today = date('Y-m-d');
+    $tomorrow = new DateTime('tomorrow');
```

Comment:
Thanks @Agares4 . This might require a thorough testing, will be done on another occasion :)